SimPEG/EM/FDEM/FieldsFDEM.py

```diff
@@ -344,11 +345,12 @@ def startup(self):
         self._MeSigma = self.survey.prob.MeSigma
         self._MeSigmaDeriv = self.survey.prob.MeSigmaDeriv
         self._MfMui = self.survey.prob.MfMui
+        self._MfMuiDeriv = self.survey.prob.MfMuiDeriv
 
     def _GLoc(self, fieldType):
         if fieldType == 'e':
```

Comment:
we don't currently have an eSecondary receiver implemented, but I guess there is no harm in allowing those fields to be retrieved.